documentation/src/docs/key-features.md

```diff
@@ -306,49 +300,46 @@ lose data is not reliable storage.  Systems based on such storage are not
-Transaction.   With Transactions, a Writer can persist data now, and later
+Transaction.  With Transactions, a Writer can persist data now, and later
 decide whether the data should be appended to a Stream or abandoned.
 
 Using a Transaction, data is written to the Stream only when the Transaction is
-committed.  When the Transaction is committed, all data written to the
-Transaction is atomically appended to the Stream.  Because Transactions are
+committed. When the Transaction is committed, all data written to the
+Transaction is atomically appended to the Stream. Because Transactions are
 implemented in the same way as Stream Segments, data written to a Transaction is
-just as durable as data written directly to a Stream.  If a Transaction is
+just as durable as data written directly to a Stream. If a Transaction is
 abandoned (e.g. if the Writer crashes) the Transaction is aborted and all data
-is discarded.  Of course, an application can choose to abort the Transaction
+is discarded. Of course, an application can choose to abort the Transaction
 through the API if a condition occurs that suggests the Writer should discard
 the data. 
 
 Transactions are key to chaining Flink jobs together.  When a Flink job uses
 Pravega as a sink, it can begin a Transaction, and if it successfully finishes
-processing, commit the Transaction, writing the data into its Pravega-based
+processing, commit the Transaction, writing the data into its Pravega based
 sink.  If the job fails for some reason, the Transaction times out and data is
 not written.  When the job is restarted, there is no "partial result" in the
 sink that would need to be managed or cleaned up.
 
 Combining Transactions and other key features of Pravega, it is possible to
-chain Flink jobs together, having one job's Pravega-based sink be the source for
-a downstream Flink job.  This provides the ability for an entire pipeline of
-Flink jobs to have end-end exactly once, guaranteed ordering of data processing.
+chain Flink jobs together, having one job's Pravega based sink be the source for
+a downstream Flink job. This provides the ability for an entire pipeline of
+Flink jobs to have end-to-end exactly once, guaranteed ordering of data processing.
 
 Of course, it is possible for Transactions across multiple Streams be
-coordinated with Transactions, so that a Flink job can use 2 or more
-Pravega-based sinks to provide source input to downstream Flink jobs.  In
+coordinated with Transactions, so that a Flink job can use two or more
+Pravega based sinks to provide source input to downstream Flink jobs.  In
 addition, it is possible for application logic to coordinate Pravega
-Transactions with external databases such as Flink's checkpoint store.
-
-Learn more about Transactions
-[here](transactions.md).
+Transactions with external databases such as Flink's checkpoint store. For more information, see [Transaction](transactions.md) section.
```

Comment:
1. Can we include security Features here:
 2. Any other new feature to be included??